setup.cfg

```diff
@@ -0,0 +1,40 @@
+[metadata]
+name = sktm
+description = "Sonic Kernel Testing"
```

Comment:
It should say "Sonic Kernel Testing Manager" isn't it?

Reply:
Fixed.